src/docs/asciidoc/reference/configuration/account-configuration.adoc

```diff
@@ -35,13 +39,6 @@ include::{docdir}/common/definitions/app-name.adoc[]
 | appName
```

Comment:
In appName row, there is an invalid sentence:
Hence ~the~ need to identify each of them -> Hence **he** need**s**